cypress/integration/sample.spec.ts

```diff
@@ -0,0 +1,10 @@
+// <reference types="cypress" />
+
+describe('example to-do app', () => {
+  it('displays two todo items by default', () => {
+    cy.visit('/app-minimal.html');
+    cy.get('div.pf-c-toolbar__item > button.pf-c-button')
+      .first()
+      .should('have.text', 'Load plugin');
+  });
+});
```

Comment:
normally we may want put tests files in `tests` folder, with more tests are added we may want another folder for common selectors

Reply:
maybe it is enough to replace `integration` with `tests`?